assets/src/scripts/components/Search.tsx

```diff
@@ -57,14 +71,14 @@ export default function Search({ draftURL, searches }: SearchProps) {
         {searches.some((search) => search.active) ? (
           <ListGroup.Item
             action
-            className="py-1 px-2 font-italic"
+            className="font-italic"
             href={encodeURI(draftURL)}
+            onClick={handleClick(draftURL)}
```

Comment:
I don't think we need the onClick handler here

Reply:
My rationale was that if you click on "show all" and don't use the onClick handler, then until the page reloads the "show all" _and_ the previous search are both highlighted in blue. Putting it here means the highlighted item updates immediately which I think looks a bit better. If we keep it like this, there's an argument to add this click logic to the included/unresolved filter card as well.

Reply:
Ah ok, that makes sense 👍 

Another side effect of us having the mixed nature of the React app and the Django app storing state.